### ISSUE TITLE
US-02: Identifying missing alternative text for images

### ISSUE THREAD

**Issue by KatjaMaximini** [KatjaMaximini/A11y-Validator#2]
As a web developer, I want to identify missing alternative text for images to make visual content accessible to screen reader users and to comply with WCAG 1.1.1 (A).
